src/content/1.7/development/multistore/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: PrestaShop multistore
+menuTitle: Multistore
+chapter: true
+aliases:
+  - /1.7/development/prestashop_multistore
+  - /1.7/development/prestashop-multistore
+---
+
+# Prestashop multistore feature
+
+PrestaShop allows you to manage multiple stores in a single PrestaShop instance: this feature is called multistore.
+
```

Comment:
```suggestion

When shop administrator turns on the Multistore option, he can configure the different shops powered by the software.
Moreover the Back Office interface is modified to allow Back Office users to manage the shop settings and content, either for all shops, for a group of shop or for a single shop.
```

Reply:
Might be somehow a duplicate of "Getting started": I let you choose if this suggestion is a good or a bad

Reply:
it's a nice addition actually